charts/pulsar/values.yaml

```diff
@@ -988,6 +988,24 @@ grafana:
     requests:
       memory: 250Mi
       cpu: 0.1
+  volumes:
+    # use a persistent volume or emptyDir
+    persistence: true
+    data:
+      name: data
+      size: 10Gi
```

Comment:
10Gi seems a lot for grafana persistence. I'd assume that 100Mi would be sufficient.

Reply:
Updated.